stock_picking_package_preparation/models/stock_picking_package_preparation.py

```diff
@@ -201,7 +201,9 @@ def _generate_pack(self):
                 picking.action_assign()
             move_lines |= picking.move_line_ids
 
-        for move_line in move_lines:
+        # If the stock.move.line has no processed quantity,
+        # set it equal to the initial demand
+        for move_line in move_lines.filtered(lambda ml: ml.qty_done == 0):
```

Comment:
@SimoRubi Careful. On fields with precision, use float_compare()

Reply:
Thanks, I didn't thought of that! Looking around a bit I found `float_is_zero` that is just what I need :) Done in https://github.com/OCA/stock-logistics-workflow/pull/448/commits/169ab09cecbf3762a29dcb670bf29971fe120209